stm32/unittest/scripts/tst_api_demod_setup

```diff
@@ -85,6 +85,66 @@ case "${TEST_OPT}" in
 		> ref_gen.log 2>&1
 	;;
 
+    700E_plain_test )
+    	# Config is <mode>, <teswtframes> 
+    	echo "81000010" > stm_cfg.txt
+	#
+        # Copy N frames of a raw audio file to stm_in.raw. 
+        dd bs=1280 count=100 if=../../../../raw/hts1.raw of=spch_in.raw \
+		> setup.log 2>&1
+	freedv_tx 700E spch_in.raw stm_in.raw --testframes --txbpf 1 \
+		>> setup.log 2>&1
+	#
+        # Reference
+        freedv_rx 700E stm_in.raw ref_demod.raw -v --testframes \
+		> ref_gen.log 2>&1
+	;;
+
+    700E_AWGN_test )
+    	# Config is <mode>, <teswtframes> 
+    	echo "81000010" > stm_cfg.txt
+	#
+        # Copy N frames of a raw audio file to stm_in.raw. 
+        dd bs=1280 count=96 if=../../../../raw/hts1.raw of=spch_in.raw \
+		> setup.log 2>&1
+	freedv_tx 700E spch_in.raw mod_bits.raw --testframes --txbpf 1 \
+		  >> setup.log 2>&1
+	cohpsk_ch mod_bits.raw stm_in.raw -20 --Fs 8000 -f -5  2>&1 | tee setup.log
+	
+        # Reference: - When the OFDM modem initially syncs, it often
+	#              has residual freq offset that causes abnormally
+	#              high LDPC decoder bit errors forthe first few
+	#              seconds.  This leads to a high coded BER being
+	#              reported for short duration tests.  This
+	#              settles down after a few seconds, and we get
+	#              the expected coded BER when averaging over
+	#              longer periods (e.g. 60s). However this
+	#              particular test is necessarily short due to the
+	#              slow speed of the semihosting system.  It is
+	#              therefore sufficient to check that the
+	#              performance is similar to the x86 C verison,
+	#              rather than expecting a low coded BER for a
+	#              short run.
+	
+        freedv_rx 700E stm_in.raw ref_demod.raw -v --testframes  2>&1 --discard | tee ref_gen.log
+	;;
+
+    700E_AWGN_codec )
+    	# Config is <mode>, <teswtframes> 
+    	echo "80000020" > stm_cfg.txt
+	#
+        # Copy N frames of a raw audio file to stm_in.raw. 
+        dd bs=1280 count=48 if=../../../../raw/hts1.raw of=spch_in.raw \
+		> setup.log 2>&1
+	freedv_tx 700E spch_in.raw mod_bits.raw --txbpf 1 \
+		>> setup.log 2>&1
+	#
+        # Reference - give it a hard time with some noise to exercise the LDPC codec and get us to max CPU
+	cohpsk_ch mod_bits.raw stm_in.raw -20 --Fs 8000 -f -5  2>&1 | tee setup.log
```

Comment:
Take a look at `test_OFDM_modem_700E_AWGN` in the x86 ctests, I think we use a Noise argument of -22 for 700E, ie a few dB higher to account for the smaller code, and higher overhead to due  increased pilot symbol rate.

Reply:
Yep, looks like -22 works. 👍